bundle-size/api.js

```diff
@@ -110,15 +111,19 @@ function erroredCheckOutput(partialBaseSha) {
 function extraBundleSizesSummary(
   headSha,
   baseSha,
+  mergeSha,
   bundleSizeDeltasRequireApproval,
   bundleSizeDeltasAutoApproved,
   missingBundleSizes
 ) {
+  const compareUrl =
+    mergeSha ||
+    `https://github.com/ampproject/amphtml/compare/${baseSha}..${headSha}`;
```

Comment:
If we're insisting on the presence of `mergeSha` in the report further down, why not just use `mergeSha`? (especially since the constructed URL is wrong more often than not...)

Reply:
Sure.